src/App.tsx

```diff
@@ -17,20 +14,11 @@ const SplashPage = styled.div`
 `;
 
 export default function App() {
-    const [allAmiibo, ,] = useState(false);
-    // Selecting amiibo data from the redux store...
-    const amiiboDataRedux = useAppSelector((state) => state.allAmiiboSlice.amiibos);
-    // console.log('Testing...: ', amiiboDataRedux);
-
-    // Filter amiibo data based on allAmiibo state
-    const filterAmiiboAllOrRecent = allAmiibo
-        ? amiiboDataRedux
-        : filterRecentReleases(amiiboDataRedux);
+    useAuthState();
```

Comment:
How to handle user session? (Part 2)

- useAuthState hook is then called in our application's root file to set up an authentication state listener. This listener will monitor for any changes in the user's authentication state throughout the lifetime of our application.

I think that's it!